src/util.rs

```diff
@@ -6,12 +6,10 @@ use super::Error;
 
 
 /// If the specified 32-bit float is a signaling NaN, make it a quiet NaN.
-///
-/// Based on [`f32::from_bits()`](https://github.com/rust-lang/rust/pull/39271/files#diff-f60977ab00fd9ea9ba7ac918e12a8f42R1279)
```

Comment:
Weeeeeeeeeeeell, these funxions are still *based on* `f{32,64}::from_bits()` in the way that they're a blatant copy-paste, so it'd be good to preserve that comment as a why/how-this-works, ya feel?

Reply:
I'm not sure that we need the statement, actually. The latest version of `from_bits` doesn't touch the bits in any way, so a suggestion that it's based on `from_bits` would give the false impression of what this last function does.

What we _can_ do is keep the comment and make it clear that it belonged to an _old_ version of `from_bits`. It'll be but a moment.